pandas/core/window/expanding.py

```diff
@@ -57,7 +58,15 @@ class Expanding(_Rolling_and_Expanding):
 
     _attributes = ["min_periods", "center", "axis"]
 
-    def __init__(self, obj, min_periods=1, center=False, axis=0, **kwargs):
+    def __init__(self, obj, min_periods=1, center=None, axis=0, **kwargs):
+        if center is not None:
+            warnings.warn(
+                "The `center` argument on `expanding` will be removed in the future",
+                FutureWarning,
+                stacklevel=3,
+            )
+        else:
+            center = False
```

Comment:
will this change break current tests when `center=True`

Reply:
It shouldn't. In that case the first branch will hit, leaving `center` unaltered and only throwing the warning

Reply:
It shouldn't. In that case `center is not None` is true, meaning center will be kept unaltered.